code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -197,63 +242,87 @@
 //nuke op pinpointers//
 ///////////////////////
 /obj/item/pinpointer/nukeop
-	var/mode = 0	//Mode 0 locates disk, mode 1 locates the shuttle
 	var/obj/docking_port/mobile/home = null
 	slot_flags = SLOT_BELT | SLOT_PDA
+	syndicate = TRUE
 
 /obj/item/pinpointer/nukeop/attack_self(mob/user as mob)
-	if(!active)
+	if(active == 0) // We want to SSfastprocess when the pinpointer goes on, BUT, we have 2 modes that can turn it on, thus the turn SSfastprocess must be put here.
+		START_PROCESSING(SSfastprocess, src)
+	if(active == 0 && !mode)
```

Comment:
We can skip checking `active == 0` twice here:
```dm
	if(active == 0)
		START_PROCESSING(SSfastprocess, src)
		if(!mode)
			active = 1
			workdisk()
			to_chat(user, "<span class='notice'>Authentication Disk Locator active.</span>")
```

Reply:
Done